atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DdlConfig.java

```diff
@@ -42,6 +42,18 @@ public int poolSize() {
         return 64;
     }
 
+    /**
+     * If set to a positive number, this is the maximum number of concurrent tasks that may execute from a single
+     * parallel operation. The overall max concurrency is still limited by the {@link #poolSize()}. Parallel operations
+     * include multiPut, truncateTables, and getRanges (postgres only).
+     * <p>
+     * If this is 0 or negative, then no limit is enforced.
+     */
+    @Value.Default
+    public int poolQosSize() {
+        return 0;
```

Comment:
If we're okay explicitly changing behavior we could probably make this default to use poolSize() instead of be hard-coded at 0, as that is probably a more reasonable default than being unlimited.